source/_integrations/matter.markdown

```diff
@@ -25,180 +25,237 @@ ha_integration_type: integration
+- For iOS, minimum version is iOS 16 (minimal 16.3 is preferred) and the most recent version of the HA companion app.
+- For Android, minimum version is 8.1 and the most recent version of both the Google Home app and the (full) HA Companion app, downloaded from the app store.
 
-There are multiple ways to run the server:
+### When I'm trying to commission using the Android app, I get an error stating "Matter is currently unavailable"
 
-**Option 1: The official Matter Server add-on, as described above**
+See above, make sure your device meets all requirements to support Matter. Update Android to the latest version and the Google Home and Home Assistant Companion app. To quickly verify if your device meets all requirements to support Matter, on your Android device go to **Settings**>**Google**>**Devices & Sharing**. There should be an entry there for **Matter devices**.
 
-_This option is only available for Home Assistant OS (the recommended installation type) and Home Assistant Supervised installations._
+Some users have reported that an uninstall and reinstall of the Google Home app fixed this issue for them.
+Also see this [extended troubleshooting guide](https://developers.home.google.com/matter/verify-services) from Google.
 
-**Option 2: Run the Matter server yourself**
+### Unable to commission devices, it keeps giving errors or stops working randomly
 
-This option is considered a very advanced setup and only for experienced users. You can find instructions on how to run the Matter Server in the [project repository](https://github.com/home-assistant-libs/python-matter-server).
+The Matter protocol relies on (local) IPv6 and mDNS (multicast traffic) which should be able to travel freely in your network. Matter devices that use WiFi (including Thread Border routers) must be on the same LAN/VLAN as Home Assistant. Matter devices that only use Thread must be joined to Thread networks for which there is at least one border router connected to the Home Assistant LAN.
 
-_Disclaimer: Some links on this page are affiliate links._
+If you experience any issues with the discovery of devices (e.g. initial commission keeps failing) or devices become unavailable randomly, investigate your network topology. For instance a setting on your router or Wi-Fi access point to "optimize" multicast traffic can actually harm the (discovery) traffic from Matter devices. Keep this in mind when you experience issues trying to add or control Matter devices. Protocols like Matter are designed for regular residential network setups and do not play nice with enterprise networking solutions like VLAN's, Multicast filtering, and IGMP snooping. To avoid issues, try to keep your network as simple and flat as possible.
```

Comment:
```suggestion
If you experience any issues with the discovery of devices (e.g. initial commission keeps failing) or devices become unavailable randomly, investigate your network topology. For instance, a setting on your router or Wi-Fi access point to "optimize" multicast traffic can actually harm the (discovery) traffic from Matter devices. Keep this in mind when you experience issues trying to add or control Matter devices. Protocols like Matter are designed for regular residential network setups and do not play nice with enterprise networking solutions like VLAN's, Multicast filtering, and IGMP snooping. To avoid issues, try to keep your network as simple and flat as possible.
```